frontend/src/components/Pages/MissionPage/MapPosition/MissionMapView.tsx

```diff
@@ -7,25 +7,31 @@ import NoMap from 'mediaAssets/NoMap.png'
 import { PlaceRobotInMap, PlaceTagsInMap } from '../../../../utils/MapMarkers'
 import { BackendAPICaller } from 'api/ApiCaller'
 import { TaskStatus } from 'models/Task'
+import { MapCompass } from 'utils/MapCompass'
 
 interface MissionProps {
     mission: Mission
 }
 
 const MapCard = styled(Card)`
     display: flex;
-    height: 600px;
-    width: 600px;
+    max-width: 600px;
     padding: 16px;
 `
 
 const StyledMap = styled.canvas`
     object-fit: contain;
-    max-height: 100%;
-    max-width: 100%;
+    max-height: 600px;
```

Comment:
I agree that ideally we want to limit px references as much as possible since these don't translate well to mobile formats, but since MapCard already has hardcoded pixel dimensions this is fine in this PR.